README.md

```diff
@@ -1,11 +1,63 @@
 ## `service-catalog`
 
-The service-catalog presents APIs for:
-
-1.  Determining which services and recipes for running instances of services
-    are available in a cluster
-2.  Making those services usable in namespaces and more granular subjects
-    (deployments, pods) within a Kubernetes cluster
+### Terminology
+
+Application: code that will access a Service. While in Kubernetes the code that
+is deployed is often called a "Service", to avoid confusion, this document will
+refer to the code that accesses a service as an "application".
+
+Recipe: the definition of an application or service in a form that can then be
+deployed into a cloud provider. For example, in Kubernetes terms, a recipe
+might be a [Deployment](http://kubernetes.io/docs/user-guide/deployments/)
+yaml file.
+
+Service: running code that is made available for use by an application.
```

Comment:
We've used "Requestable Service" at Deis to try to help clarify the difference between "Kubernetes Service" and "the thing you want to get/access/use".